modules/nodes-secondary-scheduler-about.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/secondary_scheduler/index.adoc
+
+:_content-type: CONCEPT
+[id="nodes-secondary-scheduler-about_{context}"]
+= About the {secondary-scheduler-operator}
+
+The {secondary-scheduler-operator-full} provides a way to deploy a custom secondary scheduler in {product-title}. The secondary scheduler runs alongside the default scheduler to schedule pods. Pod configurations can specify which scheduler to use.
+
+The custom scheduler must have the `/bin/kube-scheduler` binary and be based on the link:https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/[Kubernetes scheduling framework].
+
+[IMPORTANT]
+====
+You can use the {secondary-scheduler-operator} to deploy a custom secondary scheduler in {product-title}, but Red Hat does not directly support the functionality of the custom scheduler.
```

Comment:
@bergerhoffer should we say custom secondary scheduler  instead of custom scheduler may be ?

Reply:
Updated!